.gitignore

```diff
@@ -6,3 +6,4 @@
 .DS_Store
 /build/
 /.gradle/
+*.gradle
```

Comment:
what does line 9 do different than line 8?

Reply:
The builds executed inside the subprojects seem to have their own Gradle cache, so I had to ignore those